src/main/java/io/seqera/tower/cli/commands/AbstractApiCmd.java

```diff
@@ -95,25 +95,25 @@ protected void performAdditionalClientConfiguration(ClientConfig clientConfig) {
         };
     }
 
-    protected Long workspaceId() throws ApiException {
-        if (workspaceId == null) {
-            if (app().workspaceId != null) {
-                workspaceId = app().workspaceId;
-            } else {
-                if (app().orgAndWorkspaceNames != null) {
-                    loadOrgAndWorkspaceFromNames();
-                }
-            }
-        }
-        return workspaceId;
-    }
-
-    protected Long orgId() throws ApiException {
+//    protected Long workspaceId() throws ApiException {
```

Comment:
any reason why the code is commented? If it is unused and you don't plan to revert this change just delete it.

Reply:
Sorry...was there as a preventive measure but it is removed